tough/src/editor/mod.rs

```diff
@@ -289,38 +530,19 @@ impl RepositoryEditor {
 
     /// Build the `Targets` struct
     fn build_targets(&self) -> Result<Targets> {
-        let version = self.targets_version.context(error::Missing {
-            field: "targets version",
-        })?;
-        let expires = self.targets_expires.context(error::Missing {
-            field: "targets expiration",
-        })?;
-
-        // BEWARE!!! We are allowing targets to be empty! While this isn't
-        // the most common use case, it's possible this is what a user wants.
-        // If it's important to have a non-empty targets, the object can be
-        // inspected by the calling code.
-        let mut targets: HashMap<String, Target> = HashMap::new();
-        if let Some(ref existing_targets) = self.existing_targets {
-            targets.extend(existing_targets.clone());
-        }
-        if let Some(ref new_targets) = self.new_targets {
-            targets.extend(new_targets.clone());
-        }
-
-        let _extra = self.targets_extra.clone().unwrap_or_else(HashMap::new);
-        Ok(Targets {
-            spec_version: SPEC_VERSION.to_string(),
-            version,
-            expires,
-            targets,
-            _extra,
-            delegations: None,
-        })
+        Ok(self
```

Comment:
I'm really confused by this method as it's not really building the `Targets` struct. Is it necessary anymore?